README.md

```diff
@@ -68,18 +68,15 @@ Ready-to-use prompt templates for specific development scenarios and tasks, defi
 
 > 💡 **Usage**: Use `/prompt-name` in VS Code chat, run `Chat: Run Prompt` command, or hit the run button while you have a prompt open.
 
-## 🧩 Custom Chat Modes
+## 🎭 Custom Chat Modes
 
 Custom chat modes define specific behaviors and tools for GitHub Copilot Chat, enabling enhanced context-aware assistance for particular tasks or workflows.
 
-- [4.1 Beast Mode](chatmodes/4.1-Beast.chatmode.md) - A custom prompt to get GPT 4.1 to behave like a top-notch coding agent.
-- [Database Administrator Chat Mode](chatmodes/PostgreSQL%20DBA.chatmode.md) - Work with PostgreSQL databases using the PostgreSQL extension.
-- [Debug Mode Instructions](chatmodes/debug.chatmode.md) - Debug your application to find and fix a bug
-- [Planning mode instructions](chatmodes/planner.chatmode.md) - Generate an implementation plan for new features or refactoring existing code.
-- [Refine Requirement or Issue Chat Mode](chatmodes/refine-issue.chatmode.md) - Refine the requirement or issue with Acceptance Criteria, Technical Considerations, Edge Cases, and NFRs
-
-
-> 💡 **Usage**: Create new chat modes using the command `Chat: Configure Chat Modes...`, then switch your chat mode in the Chat input from _Agent_ or _Ask_ to your own mode.
+- [4.1 Beast Mode](chatmodes/4.1-beast.chatmode.md)
+- [Debug Mode Instructions](chatmodes/debug.chatmode.md)
+- [Planning mode instructions](chatmodes/planner.chatmode.md)
+- [Database Administrator Chat Mode](chatmodes/postgresql-dba.chatmode.md)
+- [Refine Requirement or Issue Chat Mode](chatmodes/refine-issue.chatmode.md)
```

Comment:
We've lost the descriptions of these, which would be useful so people understand what they are for